libs/vm/src/module.cpp

```diff
@@ -390,15 +390,11 @@ Module::Module()
       .CreateMemberFunction("set", &IShardedState::SetFromString)
       .CreateMemberFunction("set", &IShardedState::SetFromAddress);
 
-  GetClassInterface<Fixed128>().CreateSerializeDefaultConstructor(
-      [](VM *vm, TypeId) -> Ptr<Fixed128> {
+  GetClassInterface<Fixed128>()
+      .CreateSerializeDefaultConstructor([](VM *vm, TypeId) -> Ptr<Fixed128> {
         return Ptr<Fixed128>{new Fixed128(vm, fixed_point::fp128_t::_0)};
-      });
-
-  // GetClassInterface<UInt256Wrapper>().CreateSerializeDefaultConstructor(
```

Comment:
Why is there not one for UInt256?

Reply:
It is done in ledger/libs/vm-modules/src/math/bignumber.cpp:68